source/application/GUI/DlgBase.cpp

```diff
@@ -202,10 +202,10 @@ void DlgBase::AddPushButton(const std::wstring& caption,
                             DWORD id,
                             SHORT x, SHORT y,
                             SHORT w, SHORT h,
-                            bool default)
+                            bool default_)

```

Comment:
I'm not too hot about this `_`. maybe call it `default_choice` or something along those lines.